src/web/services/timezone.service.ts

```diff
@@ -109,5 +109,4 @@ export class TimezoneService {
 
     return inst.toDate().getTime();
   }
-
```

Comment:
Over here, I'd prefer to keep what you'd done before actually! We can throw an error if there's an issue. Expecting this method's users to know that it could potentially fail with a `NaN` value also seems like we're leaking too much internal details?

Sorry if I'd worded things poorly before -- I'd meant to say that whatever error message is thrown here, whoever catches it doesn't have to make use of the `error.message` as part of the toast message to be displayed, they can also write their own message instead

Reply:
Can I clarify that what you mean to catch the error and display error message in this method?

Reply:
@halfwhole in light of #11562 which adds readonly to the datatimepicker, this issue would be solved as well I assume?

Reply:
The session edit page would be fixed by those changes, but the audit logs page doesn't use the same datepicker component so it would still be editable

Reply:
Okay I see!